pandas-stubs/core/series.pyi

```diff
@@ -2076,24 +2076,41 @@ class Series(IndexOpsMixin[S1], NDFrame):
         numeric_only: _bool = ...,
         **kwargs,
     ) -> Scalar: ...
+    # Rename axis with `mapper`, `axis`, and `inplace=True`
     @overload
     def rename_axis(
         self,
-        mapper: Scalar | ListLike = ...,
-        index: Scalar | ListLike | Callable | dict | None = ...,
-        columns: Scalar | ListLike | Callable | dict | None = ...,
+        mapper: Scalar | ListLike | None = ...,
+        *,
         axis: AxisIndex | None = ...,
         copy: _bool = ...,
+        inplace: Literal[True],
+    ) -> None: ...
+    # Rename axis with `mapper`, `axis`, and `inplace=False`
+    @overload
+    def rename_axis(
+        self,
+        mapper: Scalar | ListLike | None = ...,
         *,
+        axis: AxisIndex | None = ...,
+        copy: _bool = ...,
+        inplace: Literal[False] = ...,
+    ) -> Self: ...
+    # Rename axis with `index` and `inplace=True`
+    @overload
+    def rename_axis(
+        self,
+        *,
+        index: Scalar | ListLike | Callable | dict | None = ...,
+        copy: _bool = ...,
         inplace: Literal[True],
```

Comment:
The diff is a little confusing here, but I followed the ordering in `frame.pyi`.
The ordering in `series.pyi` is currently:
```
    # Rename axis with `mapper`, `axis`, and `inplace=True`
    # Rename axis with `mapper`, `axis`, and `inplace=False`
    # Rename axis with `index` and `inplace=True`
    # Rename axis with `index` and `inplace=False`
```
Since your comment suggests swapping the second and third, I'm guessing the current ordering just wasn't clear from the diff?

Reply:
OK, thanks for confirming. No problem.